src/main/java/com/limechain/network/protocol/grandpa/GrandpaEngine.java

```diff
@@ -103,6 +123,12 @@ private void handleCommitMessage(byte[] message, PeerId peerId) {
         syncedState.syncCommit(commitMessage, peerId);
     }
 
+    /**
+     * Send our GRANDPA handshake on given stream.
```

Comment:
Please be explicit about whether the stream should be responder/initiator.
```suggestion
     * Send our GRANDPA handshake on a given <b>initiator</b> stream.
```